service/worker/migration/activities.go

```diff
@@ -470,6 +134,9 @@ func (a *activities) checkReplicationOnce(ctx context.Context, waitRequest waitR
 				tag.NewDurationTag("AllowedLagging", waitRequest.AllowedLagging),
 				tag.NewDurationTag("ActualLagging", shard.ShardLocalTime.Sub(*clusterInfo.AckedTaskVisibilityTime)),
 				tag.NewStringTag("RemoteCluster", waitRequest.RemoteCluster),
+				tag.NewInt64("MaxReplicationTaskId", shard.MaxReplicationTaskId),
+				tag.NewTimeTag("ShardLocalTime", *shard.ShardLocalTime),
```

Comment:
can you trace AllowedLaggingTasks (we are anyways also tracing AllowedLagging). Also might be easier to debug if we also trace the subtraction of MaxTaskID and AckedTaskID (slightly less math someone has to perform :))